client/src/components/Register.js

```diff
@@ -77,7 +84,15 @@ class Register extends React.Component {
             I am a volunteer
           </label>
           <label className="terms">By clicking Register, you agree to the site
-            <Link to="/terms"> terms</Link>
+            <Link onClick={this.handleOpen}> terms</Link>
```

Comment:
Can we make this link tab accessible? If `Link`s don't support [tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) we could use a regular `a` tag. Now that I think about it actually, I think my personal preference would be to actually have the `Link` update the url (probably as a query parameter) and then the modal is displayed based on the prop from the router query parameter. The close button could then remove that url query parameter. That way we would have an actual url for the terms still.